Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -78,8 +81,32 @@ static void CreateConfigFile()
                 EditorPrefs.SetBool(configFileEditorPref, true);
             }
         }
+
+        protected static ListRequest ListInstalledPackagesRequest;
+
+        [InitializeOnLoadMethod]
+        static void StoreSDKVersion()
+        {
+            ListInstalledPackagesRequest = Client.List();
+            EditorApplication.update += ListRequestProgress;
+        }
+
+        static void ListRequestProgress()
+        {
+            if (ListInstalledPackagesRequest.IsCompleted)
+            {
+                EditorApplication.update -= ListRequestProgress;
+                foreach (var package in ListInstalledPackagesRequest.Result)
+                {
+                    if (package.name.Equals("com.lootlocker.lootlockersdk"))
+                    {
+                        LootLockerConfig.current.sdk_version = package.version;
+                    }
+                }
+            }
```

Comment:
You need to unsubscribe from EditorApplication.update. So add a EditorApplication.update -= ListRequestProgress; here.